.github/workflows/cd-pypi.yml

```diff
@@ -91,9 +107,67 @@ jobs:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.PYPI_TEST_API_TOKEN }}
         run: |
-          python -m build
+          python -m build ${{ inputs.build_args }}
           twine check dist/*
           # HINT: if your upload fails here due to "unsupported local version",
           # put `local_scheme = "no-local-version"` pyproject.toml's
           # [tools.setuptools_scm]
           twine upload --repository testpypi --verbose dist/*
+
+
+  deploy_wheels:
+    if:  inputs.platforms != '' && inputs.pyversions != ''
+    needs: deploy_source
+    name: Build wheels and source distribution
+    strategy:
+      matrix:
+        os: ${{ fromJson(inputs.platforms) }}
```

Comment:
nit: if we iterate over platforms, why not name the variable platform, instead of os?